platforms/hyperledger-indy/configuration/deploy-network.yaml

```diff
@@ -36,11 +36,27 @@
       poolGenesis: "{{ poolGenesis }}"
     loop: "{{ network['organizations'] }}"
     loop_control:
-      loop_var: organizationItem  
+      loop_var: organizationItem
     when: network['type'] == 'indy' and organizationItem.type == 'peer'
 
   # delete build directory
   - name: Remove build directory
     file:
       path: "./build"
       state: absent
+
+  # Generate indy crypto and insert into Vault
+  - name: 'Generate indy crypto and insert into Vault'
```

Comment:
Wrong order, crypto has to be generated before node startup! Please follow https://alm.accenture.com/wiki/display/BLOCKOFZ/Sequence+Diagrams